{{ cookiecutter.project_name_dashed }}/.github/workflows/pr_validation.yaml

```diff
@@ -10,7 +10,11 @@ jobs:
   detect-unresolved-conflicts:
     name: Detect unresolved merge conflicts
     runs-on: ubuntu-latest
-    needs: semantic_pr
+    if: >
+      github.event.action == 'opened' ||
+      github.event.action == 'synchronize' ||
+      github.event.action == 'reopened' ||
+      github.event.action == 'ready_for_review'
```

Comment:
Wouldn't it be easier to check if it's **not** `edited`? I guess in all other conditions you run this, no?